executor/executor_test.go

```diff
@@ -4414,6 +4414,52 @@ func (s *testSuiteP2) TestUnsignedFeedback(c *C) {
 	c.Assert(result.Rows()[2][3], Equals, "table:t, range:[0,+inf], keep order:false")
 }
 
+func (s *testSuiteP2) TestShowTableNextID(c *C) {
```

Comment:
The test is a lot redundant with `TestAdminShowNextID`, how about merge those 2 test to 1.

Reply:
@zhang555 PTAL

Reply:
@crazycs520  done.